neurots/generate/orientations.py

```diff
@@ -260,7 +260,7 @@ def _mode_normal_pia_constraint(self, values_dict, tree_type):
 
             phis = self._rng.uniform(0, 2 * np.pi, len(means))
             angles += spherical_angles_to_pia_orientations(
-                phis, thetas, self._parameters.get("pia_direction", None)
+                phis, thetas, self._parameters[tree_type].get("pia_direction", None)
```

Comment:
Why is this change needed?

Reply:
yes, this one is weird, I vaguely remember I changed something, but I can't find in on region-grower, let me revert